input/cql/Status.cql

```diff
@@ -74,7 +84,12 @@ define fluent function isProcedurePerformed(Proc List<Procedure>):
     where P.status ~ 'completed'
 
 //Laboratory Test, Performed
-define fluent function isLaboratoryTestPerformed(Obs List<LaboratoryResultObservation>):
```

Comment:
According to the [mapping](https://hl7.org/fhir/us/qicore/STU6/qdm-to-qicore.html#laboratory-test-performed), this should be Laboratory Result Observation Profile